pkg/apis/operator/v1alpha1/zz_generated.openapi.go

```diff
@@ -26,15 +26,15 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"./pkg/apis/operator/v1alpha1.MetaOperatorCatalog":       schema_pkg_apis_operator_v1alpha1_MetaOperatorCatalog(ref),
-		"./pkg/apis/operator/v1alpha1.MetaOperatorCatalogSpec":   schema_pkg_apis_operator_v1alpha1_MetaOperatorCatalogSpec(ref),
-		"./pkg/apis/operator/v1alpha1.MetaOperatorCatalogStatus": schema_pkg_apis_operator_v1alpha1_MetaOperatorCatalogStatus(ref),
-		"./pkg/apis/operator/v1alpha1.MetaOperatorConfig":        schema_pkg_apis_operator_v1alpha1_MetaOperatorConfig(ref),
-		"./pkg/apis/operator/v1alpha1.MetaOperatorConfigSpec":    schema_pkg_apis_operator_v1alpha1_MetaOperatorConfigSpec(ref),
-		"./pkg/apis/operator/v1alpha1.MetaOperatorConfigStatus":  schema_pkg_apis_operator_v1alpha1_MetaOperatorConfigStatus(ref),
-		"./pkg/apis/operator/v1alpha1.MetaOperatorSet":           schema_pkg_apis_operator_v1alpha1_MetaOperatorSet(ref),
-		"./pkg/apis/operator/v1alpha1.MetaOperatorSetSpec":       schema_pkg_apis_operator_v1alpha1_MetaOperatorSetSpec(ref),
-		"./pkg/apis/operator/v1alpha1.MetaOperatorSetStatus":     schema_pkg_apis_operator_v1alpha1_MetaOperatorSetStatus(ref),
+		"github.com/IBM/meta-operator/pkg/apis/operator/v1alpha1.MetaOperatorCatalog":       schema_pkg_apis_operator_v1alpha1_MetaOperatorCatalog(ref),
+		"github.com/IBM/meta-operator/pkg/apis/operator/v1alpha1.MetaOperatorCatalogSpec":   schema_pkg_apis_operator_v1alpha1_MetaOperatorCatalogSpec(ref),
+		"github.com/IBM/meta-operator/pkg/apis/operator/v1alpha1.MetaOperatorCatalogStatus": schema_pkg_apis_operator_v1alpha1_MetaOperatorCatalogStatus(ref),
```

Comment:
seems we use different operator-sdk version which makes this different change.

My operator-sdk removed `github.com/IBM/meta-operator` and your operator-sdk added `github.com/IBM/meta-operator`, this is an endless change.

Can you use the latest stable version of operator-sdk?

Reply:
@chenzhiwei this issue don't cause by operator-sdk version, `github.com/IBM/meta-operato` removed because your project doesn't under the `GOPATH`.